client/stylesheets/_responsive.scss

```diff
@@ -162,7 +235,13 @@
 
 }
 
-@media (max-width: 720px) {
+@media (max-width: 740px) {
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens